server.js

```diff
@@ -26,160 +23,14 @@ let db = mongoose.connection;
 
 db.once("open", () => console.log("connected to the database"));
 
-// checks if connection with the database is successful
 db.on("error", console.error.bind(console, "MongoDB connection error:"));
```

Comment:
Why making things more complex?
```suggestion
db.on("error", () => console.error("MongoDB connection error:"));
```